tests/test_allure_reporter_integration.py

```diff
@@ -275,3 +276,113 @@ async def test_scenario_labels(*, dispatcher: Dispatcher, director: DirectorPlug
         ]
         assert logger.test_containers == []
         assert logger.attachments == {}
+
+
+async def test_arg_parsed_event_allure_labels(*, dispatcher: Dispatcher,
+                                              director: DirectorPlugin,
+                                              reporter: AllureReporterPlugin):
+    with given:
+        await choose_reporter(dispatcher, director, reporter)
+
+        labels = "key1=value1,key2=value2"
+        report_dir = "allure_reports"
+        args = make_parsed_args(allure_labels=labels, allure_report_dir=report_dir)
+        event = ArgParsedEvent(args)
+
+    with when:
+        await dispatcher.fire(event)
+
+    with then:
+        assert reporter._allure_labels == labels
```

Comment:
It's generally recommended to refrain from utilizing private methods within test cases, as it's not typically seen as good practice